dali/operators/reader/video_reader_op.h

```diff
@@ -66,12 +66,6 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
     DALI_ENFORCE(dtype_ == DALI_FLOAT || dtype_ == DALI_UINT8, "Data type must be FLOAT or UINT8.");
 
     enable_label_output_ = !file_root_.empty() || !file_list_.empty();
-    DALI_ENFORCE(enable_label_output_ || !enable_frame_num_,
```

Comment:
Is there any reason why this restriction was applied?

Reply:
No idea. @a-sansanwal ?

Reply:
Just making sure. If everything just works after lifting them, then fine

Reply:
I have added a new test to check that, old tests passes. No idea how I can check more.

Reply:
Yeah, that should be enough

Reply:
If I recall correctly in case of "filenames" as input, timestamp and frame number outputs don't make much sense. Because filenames provides no label output, there is no way distinguish what file it came from.

Removing this check and outputting frame numbers and timestamps is harmless but not really useful.
Or we can break existing filenames API and provide label outputs like we do for "file_root" and "file_list", then it will be meaningful.

Reply:
Good point, we may add just a consecutive numbers as a labels to each file name, then the user will know what to expect. Will update the PR>

Reply:
BTW - the changes now enables this outputs not for filenames, but for file root where you have labels assigned to directories.
But definitely it makes more sense to do this for the list of files where you can predict labels assigned to files, than file_root where it is assigned internally in DALI.

Reply:
Reworked.